src/Main.js

```diff
@@ -7,18 +7,20 @@ import { Link } from "react-router-dom";
 import Logout from "./pages/Logout";
 import { Navigation } from "./components/Navigation";
 import {ProductListing} from "./pages/ProductListing";
+import MockAPI from "./pages/Mockapi";
 
 export default function Main() {
   return (
-    <div className="MainComp">
+    <>
       <Navigation />
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/cart" element={<Cart />} />
         <Route path="/wishlist" element={<Wishlist />} />
         <Route path="/logout" element={<Logout />} />
         <Route path="/productlisting" element={<ProductListing/>} />
+        <Route path="/mockman" element={<MockAPI/>} />
       </Routes>
-    </div> 
+      </>
```

Comment:
suggestion: You can provide these routes directly in app.js itself. No need to create a Main.js component.